dist/api_data.py

```diff
@@ -1348,6 +1356,10 @@ def __cmp__(self, other):
         if read timestamp is earlier than oldest timestamp,
         read timestamp will be rounded to oldest timestamp''',
         type='boolean'),
+    Config('round_to_prepare', 'false', r'''
+        if commit timestamp is earlier than prepare timestamp,
+        commit timestamp will be rounded to prepare timestamp''',
+        type='boolean'),
```

Comment:
I think my preference is to have all the `round_to` options appear only in the `timestamp_transaction` API, and to change the name of the new prepare `round_to_oldest` to be `round_prepare_to_oldest`. Was there a request for the new options to be available at prepare/commit time?

I also think the wording for these options should be more descriptive - setting the options alters the visibility expected for a transactions updates in a way that isn't expected normally, I think we should spell that out in the descriptions here.

Reply:
I would be okay with a pointer to the docs for more description of the visibility changes instead of a long explanation in the API itself. Either way. Something like:
`This setting alters the visibility expected in a transaction. Please read <doc pointer> for a detailed explanation of the visibility effects.`